addons/repair/stringtable.xml

```diff
@@ -901,5 +901,11 @@
             <Polish>Ilość wybranych części zamiennych.</Polish>
             <Portuguese>Número de partes sobressalentes.</Portuguese>
         </Key>
+        <Key ID="STR_ACE_Repair_wheelRepairRequiredItems_name">
+            <English>Req Items for wheel repair</English>
```

Comment:
I'd maybe make this something like "Wheel repair requirements", cleaner and actually 1 character less if space is an issue